homeassistant/components/wemo/__init__.py

```diff
@@ -285,3 +345,10 @@ def validate_static_config(host: str, port: int | None) -> pywemo.WeMoDevice | N
         return None
 
     return device
+
+
+@callback
+def _async_wemo_data(hass: HomeAssistant) -> WemoData:
+    wemo_data = hass.data[DOMAIN]
+    assert isinstance(wemo_data, WemoData)
+    return wemo_data
```

Comment:
```suggestion
    return cast(WemoData, hass.data[DOMAIN])
```

Reply:
Done in [cfde9bb](https://github.com/home-assistant/core/pull/96620/commits/cfde9bbe83ce428eed16a5bf1c590c4c764c8a02)